routes/api-routes.js

```diff
@@ -135,14 +135,14 @@ module.exports = (app) => {
     logger.debug('api/claim-publish body:', body);
     logger.debug('api/claim-publish files:', files);
     // define variables
-    let  name, fileName, filePath, fileType, nsfw, license, title, description, thumbnail, channelName, channelPassword;
+    let  name, fileName, filePath, fileType, thumbnailFileName, thumbnailFilePath, thumbnailFileType, nsfw, license, title, description, thumbnail, channelName, channelPassword;
```

Comment:
Why does spee.ch need to know the `thumbnailFilePath` on the server?

Reply:
This is because I'm having the client send the the thumbnail file along with the main video file together from the front end and creating the thumbnail data dynamically (if needed) based off of the main video file's metadata on the back end.